packages/trader/src/sass/app/modules/trading.scss

```diff
@@ -605,7 +605,7 @@
         flex-direction: column;
 
         &-dropdown {
-            margin-top: 0.8rem;
+            margin-top: 0.8rem !important;
```

Comment:
@henry-deriv  another style is overriding it in packages/components/src/components/dropdown/dropdown.scss:
```
&:not(.cfd-personal-details-modal__form *) {
            margin-top: unset;
        }
```
       please add another not to it, e.g. :not(.trade_container *) to exclude dtrader page from the classnames affected by this style